runtimes/alibi-explain/mlserver_alibi_explain/runtime.py

```diff
@@ -46,6 +51,23 @@ def __init__(
         self.alibi_explain_settings = explainer_settings
         super().__init__(settings)
 
+    async def explain_v1_output(self, request: Request) -> Response:
+        """
+        A custom endpoint to return explanation results in plain json format (no v2
+        encoding) to keep backward compatibility of legacy downstream users.
+
+        This does not work with multi-model serving as no reference to the model exists
+        in the endpoint.
+        """
+        # convert raw to v2
+        raw_data = await request.body()
+        payload = InferenceRequest.parse_raw(raw_data)
+
+        v2_response = await self.predict(payload)
+        explanation = orjson.dumps(json.loads(v2_response.outputs[0].data.__root__))
+
+        return Response(content=explanation, media_type="application/json")
```

Comment:
As `Response`, we could use an `mlserver.responses.Response` object (or an ORJSONResponse object from FastAPI directly, given that `orjson` will be present). This should take care of the extra `orjson.dumps` step.

Reply:
done